code/modules/paperwork/photocopier.dm

```diff
@@ -12,8 +15,10 @@
 	var/obj/item/photo/photocopy = null
 	var/obj/item/paper_bundle/bundle = null
 	var/copies = 1 //how many copies to print!
```

Comment:
```suggestion
	///how many copies to print
	var/copies = 1
```

If you're going to add comments to the variables, you might as well do them as autodoc comments (///). See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md for more information.